wordpress/composer.json

```diff
@@ -29,7 +37,9 @@
     "wpackagist-plugin/wordpress-importer":"0.6.4",
     "wpackagist-plugin/wordpress-seo": "^16.9",
     "humanmade/s3-uploads": "^3.0",
-    "wpackagist-plugin/gutenberg":"11.9.1"
+    "wpackagist-plugin/gutenberg":"11.9.1",
+    "cds-snc/cds-base": "@dev",
+    "cds-snc/cds-default": "@dev"
```

Comment:
Treat our plugin/theme like any other composer-loaded theme/plugin, this is what gets all our dependencies in one place (see also the "path" entries in the "repositories" section above)